test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/Fixtures.html
+
+one:
+  email: MyString
+  password_digest: <%= BCrypt::Password.create('secret') %>
```

Comment:
Well, if I didn't do it, and you guys didn't do it, then probably yes :) some Rails magic right there